src/components/Display.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = (props) => {
+  let { result } = props;
+  if (result === undefined) {
+    result = '0';
+  }
+  return (
+    <div>
+      <input value={result} style={{ textAlign: 'right' }} readOnly />
+    </div>
+  );
+};
+// Display.defaultProps = {
+//   result: '0',
+// };
+Display.propTypes = {
+  result: PropTypes.string.isRequired,
```

Comment:
Let's use `defaultProps` and remove `isRequired` from here as suggested by @SunnyAsar .

The problem with the current approach is that it throws a prop validation error when we don't pass a `result` prop the component.

Reply:
If I do that, I get the following error
``` 
src/components/Display.js#L4

'result' is missing in props validation (react/prop-types)
```

Reply:
Hi @abdellani,

You should just remove the `isRequired` validator from the end but leave the rest. We need both the propTypes validation and the defaultProps:
```
Display.propTypes = {
  result: PropTypes.string,
};
Display.defaultProps = {
  result: '0',
 };
```

Reply:
Hi @zenott
Thanks for your reply.
I applied the requested changes.